README.md

```diff
@@ -74,3 +74,6 @@ Enrolment: `HMRC-PODS-ORG` `PsaId` `A2100005` (local and Staging environments on
 |Journey tests          |https://github.com/hmrc/pods-journey-tests                             |
 |Performance tests      |https://github.com/hmrc/pods-performance-tests                         |
 |Prototype              |https://github.com/hmrc/pods-prototype                                 |
+
+## Note on terminology
+The term scheme reference number and submission reference number (SRN) are interchangeable within the PODS codebase; some downstream APIs use scheme reference number, some use submission reference number, probably because of oversight on part of the technical teams who developed these APIs. This detail means the same thing, the reference number that was returned from ETMP when the scheme details were submitted.
```

Comment:
it should be 'terms' instead of 'term'

Reply:
Ah was wrong on jira ticket and I just copied it! Have amended. Will do so on scheme repos as well.